client/src/App.css

```diff
@@ -7,11 +7,15 @@ body{
   position: absolute;
   top: 50%;
   left: 50%;
-  transform: translate(-50%, -50%);
+	transform: translate(-50%, -50%);
+}
+
+.konvajs-content {
+	background: black;
 }
 
 .App {
-  text-align: center;
+	text-align: center;
```

Comment:
Indentation?

Reply:
Converting tabs to spaces.
Making a new commit to convert whole file tabs to spaces.